endpoints.json

```diff
@@ -56,5 +56,28 @@
         }
       ]
     }
-  }
+  },
+  "GET /api/articles/:articles_id/comments": {
+    "description": "serves an array of comments for the given article_id sorted by the most recent first",
+    "queries": ["article_id", "<-- article_id is 1 in this case"],
```

Comment:
is "<-- article_id is 1 in this case" a valid query? if not then it shouldn't be in the queries array!